environment.yml

```diff
@@ -8,16 +8,15 @@ channels:
   - conda-forge
 dependencies:
   - python=3.9 #need to pin to avoid issues with builds
-  - cython>=0.29.30
+  - cython>=3.0
   - compilers
   - openblas
   - nomkl
   - setuptools=59.2.0
-  - meson >= 0.64.0
   - ninja
   - pkg-config
-  - meson-python
-  - pip   # so you can use pip to install spin
+  - meson-python>=0.10.0
+  - spin=0.4
```

Comment:
no need to pin this one, I'd expect new releases to come with functionality and fixes we need